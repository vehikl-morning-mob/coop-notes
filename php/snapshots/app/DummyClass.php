<?php

namespace App;

class DummyClass
{
    private string $name;

    public function __construct(string $name)
    {
        $this->name = $name;
    }

    public function greet()
    {
        return "Hello, {$this->name}\n";
    }
}