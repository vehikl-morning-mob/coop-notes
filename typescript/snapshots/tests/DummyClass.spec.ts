import DummyClass from "../app/DummyClass";

describe('DummyClass', () => {
  it('greets', () => {
      const names = ['A', 'B', 'C', 'D', 'E'];

      let result = '';

      names.forEach(name => {
        const dummy = new DummyClass(name);
        result += dummy.greet();
      })

      expect(result).toMatchSnapshot();
  });
});