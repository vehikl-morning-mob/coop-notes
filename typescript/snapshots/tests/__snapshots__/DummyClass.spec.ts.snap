// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DummyClass greets 1`] = `
"Hello, A!
Hello, B!
Hello, C!
Hello, D!
Hello, E!
"
`;
